src/flamegraph.js

```diff
@@ -27,6 +22,44 @@ export default function () {
   var searchSum = 0
   var totalValue = 0
   var maxDelta = 0
+  var p = partition()
+
+  const containerElement = document.createElement('div')
```

Comment:
This is an anti-pattern on d3 plugins. Usually you would want to use something similar to `.append('div')` on `.enter()`.